test/NuGet.Packaging.Test/PackageFolderReaderTests.cs

```diff
@@ -11,13 +12,259 @@
 
 namespace NuGet.Packaging.Test
 {
-    public class PackageFolderReaderTests
+    public class PackageFolderReaderTests : IDisposable
     {
+        [Fact]
+        public void PackageFolderReader_NuspecCountOne()
+        {
+            // Arrange
+            var workingDir = GetTempDir();
+
+            var stream = new MemoryStream();
```

Comment:
Don't you want a using statement around the stream to dispose it. Same comment for all the new tests